mkdocs.yml

```diff
@@ -2,9 +2,10 @@ extra_css:
 - assets/versions.css
 extra_javascript:
 - assets/versions.js
-google_analytics:
-- UA-105170809-2
-- auto
+extra:
+  analytics:
+    provider: google
+    property: UA-105170809-2 # FIXME
```

Comment:
This `extra` entry needs to be moved to the start of the file (since map entries seem to be sorted and then diffed) so that the build passes.

Reply:
thanks. do you know what value should the property have?

Reply:
Yes, for the GA4 ID, see: https://github.com/argoproj/argo-site/issues/102. But it might be best to update the ID through a separate PR, WDYT? I don't have a strong preference either way.